test/common/README.md

```diff
@@ -325,6 +325,35 @@ The realpath of the 'tmp' directory.
 
 Name of the temp directory used by tests.
 
+## Fixtures Module
+
+The `common/fixtures` module provides convenience methods for working with
+files in the `test/fixtures` directory.
+
+### fixtures.fixturesDir
+
+* [&lt;String>]
+
+The absolute path to the `test/fixtures/` directory.
+
+### fixtures.fixturesPath(...args)
+
+* `...args` [&lt;String>]
+
+Returns the result of `path.join(fixturesDir, ...args)`.
```

Comment:
I just don't see the suggested bits as adding any significant value. The short bit that is there highlights the code that is being replaced by the common utility, the fact that it's an absolute path is largely irrelevant.